profiles/nis/REQUIREMENTS

```diff
@@ -4,6 +4,10 @@ Make sure that NIS service is configured and enabled. See NIS documentation for
                                                                                           {include if "with-pam-u2f"}
 - with-pam-u2f is selected, make sure that the pam u2f module is installed                {include if "with-pam-u2f"}
   - users can then configure keys using the pamu2fcfg tool                                {include if "with-pam-u2f"}
+
+          {include if "with-pam-gnome-keyring"}
```

Comment:
Please, align it to the right side.

Reply:
The requirements file(s) have been updated to remove the blank line and align the text correct.  I believe this resolves this comment.